### ISSUE TITLE
Up the hall and " " 

### ISSUE THREAD

**Issue by cranhandler** [contradb/contra#533]
"and bend into a ring" seems to be a default? I have found 2 dances in an audit of 30 where it is "up the hall and bend into a ring, right & left through" . That is not correct, it looks like 'and bend into a ring' was left selected by default. Can we default to blank so this isn't accidentally suggested?

This was done once by me, and once by u 81

**Comment by dcmorse** [comment]
It says it like 2 places "bend into a ring"!

We've got 55 dances with 'up the hall',
33 go up the hall and bend into a ring 
24 have an up the hall that doesn't end in bend-into-a-ring

Are we *really sure* we don't want bend-into-a-ring to be the default ender? It's the majority. It says it right there! It's hard to miss:

![bend](https://user-images.githubusercontent.com/10550086/47969336-91496a00-e03b-11e8-9411-7e314c6e7c29.png)

(four dances have multiple up-the-halls, hence the failure to sum perfectly)

**Comment by dcmorse** [comment]
motion to close.

**Comment by cranhandler** [comment]
No, I REALLY REALLY REALLY do not want 'bend into a ring' to be the
default. When something is a default, it gets checked when it is not meant
to be checked.

Even more than extra clicks, we do not want corrupt data.

Your response ignores the issue raised.
